ui/org.eclipse.pde.junit.runtime/META-INF/MANIFEST.MF

```diff
@@ -11,5 +11,6 @@ Require-Bundle: org.eclipse.jdt.junit.runtime;bundle-version="[3.4.0,4.0.0)",
 Export-Package: org.eclipse.pde.internal.junit.runtime;x-internal:=true
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
-Import-Package: org.eclipse.ui.testing;resolution:=optional
+Import-Package: org.eclipse.ui.testing;resolution:=optional,
+ org.junit.platform.engine
```

Comment:
The mandatory import should be avoided. In general we could also use an optional import, but since we don't need any class from that package at compile-time I suggest to use [`DynamicImport-Package`](https://docs.osgi.org/specification/osgi.core/8.0.0/framework.module.html#framework.module.dynamicimportpackage).
```suggestion
Import-Package: org.eclipse.ui.testing;resolution:=optional
DynamicImport-Package: org.junit.platform.engine
```

Reply:
Thanks for the pointer to the dynamic import option. It's perfectly reasonable here.